src/Gamepad.js

```diff
@@ -0,0 +1,68 @@
+let gamepadId = 0;
+
+export class Gamepad extends HTMLElement {
```

Comment:
I can't extend `HTMLDivElement` because it seems that it [isn't declared as a class](https://stackoverflow.com/questions/24823442/can-you-extend-a-htmldivelement-in-typescript). The Stackoverflow solution does suggest a way to extend `HTMLDivElement` via delegation. Am not sure whether this is currently necessary. Having said that, it seems that `HTMLButtonElement` also cannot be extended and perhaps we might want specific `Button` functionality when I add the gamepad buttons. 

Thoughts?